regression-test/suites/load_p0/stream_load/test_steam_load_with_data_quality.groovy

```diff
@@ -0,0 +1,187 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import java.util.Random;
+
+suite("test_stream_load_with_data_quality", "p0") {
+    if (!isCloudMode()) {
+        return;
+    }
+
+    def tableName = "test_stream_load_with_data_quality"
+    sql "DROP TABLE IF EXISTS ${tableName}"
+
+    sql """
+        CREATE TABLE ${tableName}
+        (
+            siteid INT DEFAULT '10',
+            citycode SMALLINT NOT NULL,
+            username VARCHAR(32) DEFAULT '',
+            pv BIGINT SUM DEFAULT '0'
+        )
+        AGGREGATE KEY(siteid, citycode, username)
+        DISTRIBUTED BY HASH(siteid) BUCKETS 1;
+    """
+
+    String label = UUID.randomUUID().toString().replaceAll("-", "")
+
+    // meta-service max_num_aborted_txn is 100
+    for (int i = 0; i < 100; i++) {
+        streamLoad {
```

Comment:
100 should be got from configs of be via http. otherwise case is not stable.

Reply:
> 100 should be got from configs of be via http. otherwise case is not stable.

ms not provider http api for querying. ms's config